docs/using-the-compiler.rst

```diff
@@ -287,6 +287,9 @@ Input Description
             "cse": false,
             // Optimize representation of literal numbers and strings in code.
             "constantOptimizer": false,
+            // Use unchecked arithmetic when incrementing the counter of for loops
+            // under certain circumstances. It is always on if no details are given.
```

Comment:
We should really have documented the precise conditions somewhere in the docs :-). But let's not hold things up due to that.

Reply:
There's still time to do it in a follow-up PR :)